.gitignore

```diff
@@ -0,0 +1,19 @@
+.svn/
+.hg/
+.hgtags/
+log
+*.py[cod]
+build
+doc/_build
+dist/
+pylint.egg-info/
+.tox
+*.sw[a-z]
+doc/features.rst
+pyve
+build-stamp
+debian/files
```

Comment:
The `debian` ignores you cited are not rooted.  That being said, I do see the ones that are (e.g. `build`, `doc/_build`, etc) and will update the gitignore to match those. Thanks.